islands/serviceMap.tsx

```diff
@@ -0,0 +1,53 @@
+import ReactFlow, {
```

Comment:
I renamed flow to serviceMap to try to better describe what the component means to us...flow kind of seemed to leak in implementation detail into the name.